test/index.js

```diff
@@ -84,21 +85,28 @@ describe('simple-keyring', function () {
 
     it('returns a signed tx object', async function () {
       await keyring.deserialize([privateKey]);
-
-      const txParams = {
-        from: address,
-        nonce: '0x00',
-        gasPrice: '0x09184e72a000',
-        gasLimit: '0x2710',
-        to: address,
-        value: '0x1000',
-      };
       const tx = TransactionFactory.fromTxData(txParams);
       expect(tx.isSigned()).toBe(false);
 
       const signed = await keyring.signTransaction(address, tx);
       expect(signed.isSigned()).toBe(true);
     });
+
+    it('throw error if empty address is passed', async function () {
```

Comment:
This seems slightly inaccurate. I know it's a bit misleading because you're using `toThrow`, but I think it might be more accurate to say:

``` suggestion
    it('returns rejected promise if empty address is passed', async function () {
```

Similar for the other tests here.